Makefile.am

```diff
@@ -5325,6 +5323,11 @@ endif
 
 # ------------------------------------------------------------------------------
 if ENABLE_MACHINED
+
+dist_systemunit_DATA += \
+	units/var-lib-machines.mount \
+	units/var-lib-machines.automount
```

Comment:
Same as other comment, nspawn is a mere user and as such having the mount units when nobody will create the underlying loop device is not useful.